dev/com.ibm.ws.ssl/src/com/ibm/websphere/ssl/Constants.java

```diff
@@ -212,7 +213,7 @@ private Constants() {
     public static final String IBMJCE = "com.ibm.crypto.provider.IBMJCE";
     public static final String IBMJCE_NAME = "IBMJCE";
     public static final String IBMJCEFIPS = "com.ibm.crypto.fips.provider.IBMJCEFIPS";
-    public static final String IBMJCEPlusFIPS = "com.ibm.crypto.fips.provider.IBMJCEPlusFIPS";
+    public static final String IBMJCEPlusFIPS = "com.ibm.crypto.plus.fips.provider.IBMJCEPlusFIPS";
```

Comment:
Did they change the package name or has this been wrong all along?  If they changed the package name do we need to support both of them when running with other versions of the JDK?

Reply:
This has been wrong all along.

Reply:
From what I can tell `com.ibm.crypto.provider.IBMJCEPlusFIPS` is used for 140-2 and `com.ibm.crypto.plus.provider.IBMJCEPlusFIPS` is the one for 140-3?  Is that right?  This is an API class for customers to be able to use as well.  Adding new fields to this class should have triggered a feature checker failure due to requirement to update the API bundle version, package version and javadoc version for this package.   There is also a lack of Javadoc for all of the fields as well.

Reply:
https://openliberty.io/docs/latest/reference/javadoc/api/ssl-1.0.com.ibm.websphere.ssl.html?path=24.0.0.12/com.ibm.websphere.appserver.api.ssl_1.7-javadoc/com/ibm/websphere/ssl/Constants.html shows the current state of the javadoc which is not very usable for the constants for sure.

Reply:
Looks like I may be wrong.  `com.ibm.crypto.plus.provider.IBMJCEPlusFIPS` appears to be what is used this whole time.  This really does affect the logic in `AbstractJSSEProvider` to call initializeFips.  It would appear that never would have been called, so I may be able to remove my objection to the logic that is commented out in there except to just remove it instead of comment it out.

Reply:
With the update to the Constants API, what I am saying needs to be updated is the version in the `com.ibm.websphere.appserver.api.ssl/bnd.bnd` file to `1.8` and updating of the `com.ibm.websphere.appserver.ssl-1.0.feature` file to reference the new version.  Also I believe that the `package-info.java` class for the `com.ibm.websphere.ssl` package should be updated to `1.9`.